File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/PlayerLeaveWaitingRoomMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.user.UserDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PlayerLeaveWaitingRoomMessage extends BaseMessage {
    private UserDto player;

    public PlayerLeaveWaitingRoomMessage() {
        super(MessageType.PLAYER_LEAVE_WAITING_ROOM);
    }
}

File: front/src/app/core/domain/message/player-leave-waiting-room.message.ts
import {BaseMessage} from "@domain/message/base.message";
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import {UserDto} from "@domain/user.dto";

export class PlayerLeaveWaitingRoomMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.PLAYER_LEAVE_WAITING_ROOM;
  player: UserDto;
}

